### PR TITLE
Docs 2696 add terms to glossary

### PR DESCRIPTION
**Pull request by avanscoy**

### CODE REVIEW

File: config/glossary.json [hunk 1]
```diff
@@ -29,6 +34,11 @@
     "definition": "The unique identifier of the audience for an issued token, identified within a JSON Web Token as the `aud` claim. The audience value is either the application (`Client ID`) for an ID Token or the API that is being called (`API Identifier`) for an Access Token. At Auth0, the Audience value sent in a request for an Access Token dictates whether that token is returned in an opaque or JWT format.",
     "short": "A unique identifier, <b>aud</b> in a token, for an application (ID Token) or an API (Access Token)."
   },
+  "auth0-dashboard": {
+    "title": "Auth0 Dashboard" ,
+    "definition": "Auth0’s main product in which you can register your application or API, connect to a user store or another identity provider, and configure your Auth0 services.",
```
Comment (tr0njavolta) [inline config/glossary.json]: Main product is confusing to me for some reason. Is "primary administrator interface" another way to describe the dashboard?

File: config/glossary.json [hunk 2]
```diff
@@ -44,11 +54,26 @@
...
     "short": "Product release stage during which the referemced feature or behavior is provided to subscribers to give them time to explore and adopt new product capabilities while providing final feedback prior to a General Availability (GA) release. Functionality is code-complete, stable, useful in a variety of scenarios, and believed to meet or almost meet quality expectations for a GA release. Beta releases may be restricted to a select number of subscribers (private) or open to all subscribers (public)."
   },
+  "block-unblock": {
+    "title": "Block/Unblock Users" ,
+    "definition": "Refers to features from Auth0's Attack Protection suite, Breached Password Detection, Brute-Force Protection, and Suspicious IP Throttling. Each service assesses login/sign-up trends and blocks IP addresses associated with suspicious activity.",
```
Comment (tr0njavolta) [inline config/glossary.json]:
```suggestion
    "definition": "Refers to the features from Auth0's Attack Protection suite: Breached Password Detection, Brute-Force Protection, and Suspicious IP Throttling. Each service assesses login/sign-up trends and blocks IP addresses associated with suspicious activity.",
```

Is the Attack Protection suite a separate feature from the other aspects or is it the collection of features?

**Review by tr0njavolta — DISMISSED** [review]
A few questions and comments, but approving so I don't block

File: config/glossary.json [hunk 3]
```diff
@@ -4,6 +4,11 @@
     "definition": "A credential that can be used by an application to access an API. It informs the API that the bearer of the token has been authorized to access the API and perform specific actions specified by the scope that has been granted. An Access Token can be in any format, but two popular options include opaque strings and JSON Web Tokens (JWT). They should be transmitted to the API as a Bearer credential in an HTTP Authorization header.",
     "short": "An authorization credential, in the form of an opaque string or JWT, used to access an API."
   },
+  "account linking": {
+    "title": "Account Linking",
+    "definition": "Connecting user accounts across multiple platforms to allow users access to more than one resource or application by providing credentials one time.",
+    "short": "Connecting user accounts across multiple platforms.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma at the end of this line.

File: config/glossary.json [hunk 4]
```diff
@@ -29,6 +34,11 @@
     "definition": "The unique identifier of the audience for an issued token, identified within a JSON Web Token as the `aud` claim. The audience value is either the application (`Client ID`) for an ID Token or the API that is being called (`API Identifier`) for an Access Token. At Auth0, the Audience value sent in a request for an Access Token dictates whether that token is returned in an opaque or JWT format.",
     "short": "A unique identifier, <b>aud</b> in a token, for an application (ID Token) or an API (Access Token)."
   },
+  "auth0-dashboard": {
+    "title": "Auth0 Dashboard",
+    "definition": "Auth0’s primary administrator interface in which you can register your application or API, connect to a user store or another identity provider, and configure your Auth0 services.",
+    "short": "Auth0's main product to configure your services.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma at the end of this line.

File: config/glossary.json [hunk 5]
```diff
@@ -44,11 +54,26 @@
...
     "short": "A random string generated by the authorization server and returned to the application as part of the authorization response. The authorization code is relatively short-lived and is exchanged for an Access Token at the token endpoint when using the Authorization Code Flow."
   },
+  "authorization-flow": {
+    "title": "Authorization Flow",
+    "definition": "Another name for Authorization Grants outlined in OAuth 2.0. Authorization flows are the workflows a resource (i.e. an application or an AIP) uses to grant requestors access. Based on the type of technology (i.e. if an application can store a Client Secret) and the type of requestor, resource owners can use Authorization Code Flow, Proof of Key Code Exchange (PKCE), Resource Owner Password Credential (ROPG), Implicit, or Client Credential.",
+    "short": "Authorization grant (or workflow) specified in the OAuth 2.0 framework.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma at the end of this line.

File: config/glossary.json [hunk 6]
```diff
@@ -44,11 +54,26 @@
...
+  },
+  "bad-actors": {
+    "title": "Bad Actors",
+    "definition": "Also known as threat actors. An entity (a person or group) that poses a threat to the business or environment with the intention to cause harm. Harm can constitute physical or cyber damages, from breaking into a data center to hacking into systems with stolen credentials.",
+    "short": "An entity (a person or group) that poses a threat to the business or environment with the intention to cause harm.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 7]
```diff
@@ -44,11 +54,26 @@
...
     "short": "Product release stage during which the referemced feature or behavior is provided to subscribers to give them time to explore and adopt new product capabilities while providing final feedback prior to a General Availability (GA) release. Functionality is code-complete, stable, useful in a variety of scenarios, and believed to meet or almost meet quality expectations for a GA release. Beta releases may be restricted to a select number of subscribers (private) or open to all subscribers (public)."
   },
+  "block-unblock": {
+    "title": "Block/Unblock Users",
+    "definition": "Refers to the features from Auth0's Attack Protection suite, Breached Password Detection, Brute-Force Protection, and Suspicious IP Throttling. Each service assesses login/sign-up trends and blocks IP addresses associated with suspicious activity.",
+    "short": "Removing or restoring a requestors access to a resource.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 8]
```diff
@@ -79,6 +104,11 @@
     "definition": "An attribute packaged in a security token which represents a claim that the provider of the token is making about an entity.",
     "short": "An attribute packaged in a security token which represents a claim that the provider of the token is making about an entity."
   },
+  "client-id": {
+    "title": "Client ID",
+    "definition": "Identification value assigned to your application after registration. This value is used in conjunction with other third-party services and can be found in Auth0 Dashboard > Application Settings.",
+    "short": "Identification value given to your registered resource from Auth0.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma.

File: config/glossary.json [hunk 9]
```diff
@@ -99,6 +129,12 @@
     "definition": "The relationship between Auth0 and the sources of users for your applications. Examples include identity providers (such as Google or Active Directory), passwordless authentication methods, or user databases.",
     "short": "The relationship between Auth0 and the sources of users for your applications."
   },
+  "custom-domain": {
+    "title": "Custom Domain",
+    "definition": "A third-party domain with a specialized, or vanity, name. Also known as a CNAME.",
+    "short": "A third-party domain with a specialized, or vanity, name.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 10]
```diff
@@ -134,6 +170,11 @@
     "definition": "The date that access to a feature or behavior is removed from the platform. End Of Life Dates can vary between different plan types.",
     "short": "The date that access to a feature or behavior is removed from the platform. End Of Life Dates can vary between different plan types."
   },
+  "fine-grained-auth": {
+    "title": "Fine Grained Authorization (FGA) ",
+    "definition": "Auth0’s SaaS product gives individual users access to specific objects or resources within your application.",
+    "short": "A product allowing individual users access to specific objects or resources.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 11]
```diff
@@ -164,11 +205,27 @@
     "definition": "An open, industry standard <a href='https://tools.ietf.org/html/rfc7519'>RFC 7519</a> method for representing claims securely between two parties. At Auth0, ID Tokens are always returned in JWT format, and Access Tokens are often in JWT format. You may decode well-formed JWTs at <a href='https://jwt.io'>JWT.io</a> to view their claims. See <a href='/docs/jwt'>JSON Web Tokens</a>.",
     "short": "The standard ID Token format and often Access Token format used to represent claims securely between two parties."
   },
+  "localization": {
+    "title": "Localization",
+    "definition": "Ability to render the New Universal Login experience into a supported languages.",
+    "short": "Ability to render NUL into a supported language.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 12]
```diff
@@ -164,11 +205,27 @@
...
+
   "lock": {
     "title": "Lock",
     "definition": "Auth0's UI widget for authenticating users. It is ready to go as-is and is the default face of the Classic Universal Login experience. Lock allows you to customize minor behavioral and appearance options, but its primary goal is ease of use. See <a href='/docs/lock'>Lock</a>.",
     "short": "Auth0's UI widget for authenticating users."
   },
+  "management-api": {
+    "title": "Management API",
+    "definition": "A product to allow customers to manage Auth0 services and perform administrative tasks programatically.",
+    "short": "A product to allow customers to perform administrative tasks. ",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 13]
```diff
@@ -164,11 +205,27 @@
...
+  },
+  "metadata": {
+    "title": "Metadata",
+    "definition": "Information users can update, such as preferences or profile settings. Metadata is added to ID tokens and can be stored in user profiles.",
+    "short": "User information stored in user profiles.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 14]
```diff
@@ -184,11 +241,21 @@
     "definition": "An arbitrary (often random or pseudo-random) number issued in an authentication protocol that can be used to help detect and mitigate replay attacks using old communications. In other words, the nonce is only issued once, so if an attacker attempts to replay a transaction with a different nonce, its false transaction can be detected more easily. See <a href='/docs/api-auth/tutorials/nonce'>Nonce</a>.",
     "short": "An arbitrary number issued once in an authentication protocol to detect and prevent replay attacks."
   },
+  "oath2": {
+    "title": "OAuth 2.0",
+    "definition": "An authorization framework that defines authorization protocols and workflows. OAuth 2.0 defines roles, authorization grants (or workflows), authorization requests and responses, and token handling.",
+    "short": "An authorization framework that defines authorization protocols and workflows.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 15]
```diff
@@ -184,11 +241,21 @@
...
+  },
   "openid": {
     "title": "OpenID",
     "definition": "An open standard for authentication that allows applications to verify users are who they say they are without needing to collect, store, and therefore become liable for a user’s login information. See <a href='/docs/protocols/oidc'>OpenID Connect (OIDC)</a>.",
     "short": "An open standard for authentication that allows applications to verify users' identities without collecting and storing login information." 
   },
+  "organizations": {
+    "title": "Organizations",
+    "definition": "A product for B2B customers to categorize end-users and define specific roles, login experience, and access to resources.",
+    "short": "A product to categorize end-users and define specific roles.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 16]
```diff
@@ -269,6 +336,11 @@
     "definition": "A difficult to sustain practice of manually provisioning a user from a local directory separately in a remote directory (essentially creating a copy, or shadow, of the original account) when they need access to remote applications.",
     "short": "A difficult to sustain practice of manually provisioning a user from a local directory separately in a remote directory."
   },
+  "signing-algorithm": {
+    "title": "Signing Algorithm",
+    "definition": "A hashing algorithm used to digitally sign tokens to ensure the token has not been tampered with by bad actors.",
+    "short": "An algorithm used to digitally sign tokens to ensure the token has not been tampered with.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

File: config/glossary.json [hunk 17]
```diff
@@ -308,5 +380,10 @@
    "title": "Universal Login",
    "definition": "Auth0’s implementation of the authentication flow, which is the key feature of an Authorization Server. Each time a user needs to prove their identity, your applications redirect to Universal Login, and Auth0 will do what’s needed to guarantee the user’s identity. See <a href='/docs/universal-login'>Auth0 Universal Login</a>.",
    "short": "Your application redirects to Universal Login, hosted on Auth0's Authorization Server, to verify a user's identity."
+ },
+ "ws-fed": {
+  "title": "Web Service Federation (WS-Fed)",
+  "definition": "A protocol for managing user identities between systems, domains, and identity providers with established trust using WS-Trust. This protocol is mainly used for Microsoft products and defines policies on how to share federation metadata. ",
+  "short": "A protocol for managing user identities across domains.",
```
Comment (rkhoriander) [inline config/glossary.json]: Extra comma

**Review by rkhoriander — DISMISSED** [review]

File: config/glossary.json (hunk 1 above)
Comment (avanscoy) [reply config/glossary.json]: This one has been addressed. Thanks for the help!

File: config/glossary.json (hunk 2 above)
Comment (avanscoy) [reply config/glossary.json]: You know, this is a good point, a colon here would be better. Attack Protection is the collection of features but also includes Bot Detection, which does not block users at all.  Sorry I did not see this comment earlier!

**Review by tr0njavolta — APPROVED** [review]

**Pull request merged**